hw/ip/hmac/hmac.core

```diff
@@ -3,20 +3,13 @@ CAPI=2:
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 name: "lowrisc:ip:hmac:0.1"
-description: "HMAC-SHA256 engine"
+description: "HMAC- multi-mode SHA-2 engine"
```

Comment:
I believe we should not be changing this description yet, since we haven't actually implemented multi-mode just yet.

Reply:
thanks, i updated the description in my last forced push.